core/src/Telemetry/DefaultEvent.cs

```diff
@@ -35,7 +35,7 @@ public DefaultEvent(string clientId) : base((string) (EventBase.EventNamePrefix
         {
             this[EventNamePrefix + "client_id"] = clientId;
             this[EventNamePrefix + "sdk_platform"] = CorePlatformInformationBase.Instance.GetProductName()?.ToLowerInvariant();
-            this[EventNamePrefix + "sdk_version"] = MsalIdHelper.GetMsalVersion();
+            this[EventNamePrefix + "sdk_version"] = CorePlatformInformationBase.GetClientVersion();
```

Comment:
>CorePlatformInformationBase [](start = 52, length = 27)

Keep it instance based (see my comment on the use of statics) . those are super evil!